openg2p_registration/models/openg2p_registration.py

```diff
@@ -581,6 +590,123 @@ def create_beneficiary_from_registration(self):
+                "street": str(self.street),
+                "street2": str(self.street),
+                "city": str(self.city),
+                "postal_code": str(self.zip),
+                "identity": str(self.identity_passport),
+                "bank": str(self.bank_account_id.bank_id.name),
+                "bank_account": str(self.bank_account_id.sanitized_acc_number),
+                "emergency_contact_name": str(self.emergency_contact),
+                "emergency_contact_phone": str(self.emergency_phone),
+            }
+        )
+
+        delete_url = "http://localhost:8080/index/" + str(idr)
+        r = requests.post(delete_url)
+        print(r)
+        self.clear_beneficiaries()
+        self.retained_id = 0
+
+    def clear_beneficiaries(self):
+        self.write({"duplicate_beneficiaries_ids": [(5, 0, 0)]})
+
+    def index_beneficiary(self):
+        data = {
+            "id": str(self.beneficiary_id.id),
+            "first_name": str(self.firstname),
+            "last_name": str(self.lastname),
+            "email": str(self.email),
+            "phone": str(self.phone),
+            "street": str(self.street),
+            "street2": str(self.street2),
+            "city": str(self.city),
+            "postal_code": str(self.zip),
+            "dob": str(self.birthday),
+            "identity": str(self.identity_passport),
+            "bank": str(self.bank_account_id.bank_id.name),
+            "bank_account": str(self.bank_account_id.sanitized_acc_number),
+            "emergency_contact_name": str(self.emergency_contact),
+            "emergency_contact_phone": str(self.emergency_phone),
+        }
+        # Deleting null fields
+        new_data = self.del_none(data)
+        print(new_data)
+        url_endpoint = "http://localhost:8080/index"
+        try:
+            r = requests.post(url_endpoint, json=new_data)
+            return r.status_code
+        except requests.exceptions.RequestException as e:
+            print(e)
+
+    def search_beneficiary(self):
+        print("Searching Beneficiaries.....")
+        search_data = {
+            "attributes": {
+                "first_name": str(self.firstname),
+                "last_name": str(self.lastname),
+                "email": str(self.email),
+                "phone": str(self.phone),
+                "street": str(self.street),
+                "street2": str(self.street2),
+                "city": str(self.city),
+                "postal_code": str(self.zip),
+                "dob": str(self.birthday),
+                "identity": str(self.identity_passport),
+                "bank": str(self.bank_account_id.bank_id.name),
+                "bank_account": str(self.bank_account_id.sanitized_acc_number),
+                "emergency_contact_name": str(self.emergency_contact),
+                "emergency_contact_phone": str(self.emergency_phone),
+            }
+        }
+        new_data = self.del_none(search_data)
```

Comment:
Rename these variables - new_data. my_data etc. Make it a more suitable naming.

Reply:
Sure, will rename it.